auth-web/src/components/auth/common/HelpDialog.vue

```diff
@@ -77,3 +79,21 @@ export default class HelpDialog extends Vue {
   }
 }
 </script>
+
+
+<style lang="scss" scoped>
+  .v-card {
+    > div {
+        padding: 16px 40px 0 40px;
```

Comment:
Good, this is for all of the custom fitting for the dialogs. Should we use this for all dialogs perhaps?